app/pages/index.js

```diff
@@ -137,52 +138,55 @@ const Home = ({ randomContributors }) => {
                 />
                 <Scrollbar className="lg:hidden" thumbHeight="4px">
                   <div className="grid pb-4 ml-6 features-grid lg:gap-y-14">
-                    <Feature title="Fullstack & Monolithic">
+                    <Feature title="Fullstack & monolithisch">
                       <p>
-                        Includes everything from the database to your frontend all inside a single
-                        app. Only one thing to develop. Only one thing to deploy.
+                        Bevat alles van de database tot je frontend in één app. Er is maar één ding
+                        om te ontwikkelen. Maar één ding om uit te rollen.
                       </p>
-                      <p>And you can deploy to a server or serverless.</p>
+                      <p>En je kunt het op een server of serverless uitrollen.</p>
                     </Feature>
-                    <Feature title="API Not Required">
+                    <Feature title="Geen API nodig">
                       <p>
-                        Instead of fetching data from the backend, you import your server code
-                        directly into your components. At build time, that function import is
-                        swapped out with an auto generated HTTP API.
+                        In plaats van data uit de backend op te halen, importeer je je servercode
+                        rechtstreeks in je componenten. Tijdens het compileren wordt die
```

Comment:
Op andere plaatsen heb je `components` gebruikt. Welk van de twee gaan we gebruiken?

Reply:
Oeps :) Goh, ik heb eigenlijk niet echt een voorkeur. Jij?